docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 11-14
+      :language: sql
+
+#. Run the following command to list the tables for the ``demo-flink`` service:
+
+   ::
+
+      avn service flink table list demo-flink
+
+   The output for this command shows you the table IDs, which you need in the command that you use to create Flink jobs:
+
+   ::
+
+     INTEGRATION_ID                        TABLE_ID                              TABLE_NAME
+     ====================================  ====================================  ==========
+     917bbec0-0f34-4a31-b910-c585feb95d09  305c44d9-22d5-4be8-987f-57c7642e8a89  CPU_IN
+     917bbec0-0f34-4a31-b910-c585feb95d09  3d33a7c5-3716-4b21-9739-f79228f9f28f  CPU_OUT_FILTER
+
+#. Run the following command to create a data pipeline job named ``simple_filter``:
+
+   ::
+
+      avn service flink job create demo-flink simple_filter     \
+          --table-ids CPU_IN_ID CPU_OUT_FILTER_ID               \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN`` and ``CPU_OUT_FILTER`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 17-24
+      :language: sql
+
+
+Create a pipeline with windowing
+--------------------------------
+   
+This setup uses :doc:`windows </docs/products/flink/concepts/windows>` to determine instances of high CPU load during set intervals based on :doc:`event time </docs/products/flink/concepts/event_processing_time>`.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Kafka source)-- timestamped metrics -->id3(Flink job);
+        id3-- 30-second average CPU -->id4(Kafka sink);
+
+
+1. Using the Aiven CLI, run the following command to create a Kafka table named ``CPU_OUT_AGG``:
+
+   ::
+
+      avn service flink table create demo-flink KAFKA_INTEGRATION_ID  \
+          --table-name CPU_OUT_AGG                                    \
+          --kafka-topic cpu_load_stats_agg                            \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 27-32
+      :language: sql
+
+#. Run the following command to list the tables for the ``demo-flink`` service and get the IDs for the ``CPU_IN`` and ``CPU_OUT_AGG`` tables:
+
+   ::
+
+      avn service flink table list demo-flink
+
+#. Run the following command to create a data pipeline job named ``simple_agg``:
+
+   ::
+
+      avn service flink job create demo-flink simple_agg        \
+          --table-ids CPU_IN_ID CPU_OUT_AGG_ID                  \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN`` and ``CPU_OUT_AGG`` tables, and ``JOB_SQL`` with the following:
```

Comment:
Put the instruction for how to construct or edit the command before the command itself.